### PR TITLE
Replace sprintf_P with strcpy_P where possible.

### PR DESCRIPTION
**Pull request by bubeck**
I recplaced all occurences of sprintf_P with strcpy, where no % is used. This increases performance.
Additionally the function to draw an arrow is now able to deal with any angle from -inf to +inf and wll normalize itself. Therefore the multiple normalization before calling the function is eliminated. :)

### CODE REVIEW

File: skydrop/src/gui/widgets/widgets.cpp [hunk 1]
```diff
@@ -294,6 +294,11 @@ void widgets_draw(uint8_t page)
  */
 void widget_arrow(uint16_t angle, uint8_t x, uint8_t y, uint8_t w, uint8_t h)
 {
+	// make sure, that angle is always between 0 and 359:
+	if (angle < 0 || angle > 359) {
+		angle = (angle % 360 + 360) % 360;
```
Comment (fhorinek) [inline skydrop/src/gui/widgets/widgets.cpp]: Should be angle = (angle + 360) % 360

File: skydrop/src/gui/widgets/widgets.cpp [hunk 2]
```diff
@@ -294,6 +294,11 @@ void widgets_draw(uint8_t page)
  */
 void widget_arrow(uint16_t angle, uint8_t x, uint8_t y, uint8_t w, uint8_t h)
```
Comment (fhorinek) [inline skydrop/src/gui/widgets/widgets.cpp]: Also angle should be signed then

Comment (bubeck) [inline skydrop/src/gui/widgets/widgets.cpp]: done

File: skydrop/src/gui/widgets/widgets.cpp (hunk 1 above)
Comment (bubeck) [inline skydrop/src/gui/widgets/widgets.cpp]: Done. However, my solution was able to deal with angle < -360 (which is probably never occuring).,

**Comment by fhorinek** [comment]
Can you use modulo on negative numbers?

**Comment by bubeck** [comment]
Yes, modulo is possible on negative numbers and does, what we want in this case.
See http://stackoverflow.com/questions/11720656/modulo-operation-with-negative-numbers

**Comment by fhorinek** [comment]
then I am sorry for my limited knowledge, please revert the change

**Comment by bubeck** [comment]
Are there any remaining questions? Otherwise I would send the next pull for the waiting next features... :)

**Pull request merged**